lib/bundler/installer.rb

```diff
@@ -5,6 +5,7 @@
 require "bundler/installer/parallel_installer"
 require "bundler/installer/standalone"
 require "bundler/installer/gem_installer"
+require "bundler/process_lock"
```

Comment:
please move this to an autoload under the bundler namespace in `bundler.rb`, since the file isn't in the `installer` subdirectory